src/SFA.DAS.Roatp.CourseManagement.Jobs/Infrastructure/ApiClients/StandardsApi/GetAllStandardsApiClient.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Roatp.CourseManagement.Jobs.Infrastructure.ApiClients.StandardsApi.Models;
+
+namespace SFA.DAS.Roatp.CourseManagement.Jobs.Infrastructure.ApiClients.StandardsApi
+{
+    public class GetAllStandardsApiClient : ApiClientBase<GetAllStandardsApiClient>, IGetAllStandardsApiClient
+    {
+        public GetAllStandardsApiClient(HttpClient client, ILogger<GetAllStandardsApiClient> logger)
+            : base(client, logger)
+        {
+        }
+    
+        public async Task<StandardList> GetAllStandards()
+        {
+            return await Get<StandardList>("api/courses/Standards?Filter=None");
```

Comment:
Question, can this bring back multiple records for a LarsCode with different version? if  it can then how do we mitigate this?

Reply:
Ah good spot.  We do indeed bring back more than one record from standards for some lars codes.  Eg 1. 

I'll do some analysis to explain it, and discuss with you directly, but I reckon it's probably because of retired/withdrawn versions.

Reply:
It might be Filter=None is too broad, and I can find the appropriate filter